### PR TITLE
Feature/profiles

### PR DESCRIPTION
**Pull request by Salvarez0212**
## Summary

Section of appointments in both doctor and user profile was added. Pending to link with backend for a dynamic rendering of appointmets.

image

image

### CODE REVIEW

File: src/pages/profiles/components/doctorProfile/DoctorProfile.page.jsx [hunk 1]
```diff
@@ -151,15 +153,22 @@ export const DoctorProfile = ({
...
+              variant="solid"
+              color="info"
+              className={`${editState} save__button`}
+              onClick={handleEditOff}
+            >
+              Save changes
+            </Button>
+          </article>
+          <article className="appointments__section-doctor">
+            {appointments.length > 0 ? (
```
Comment (ht1204) [inline src/pages/profiles/components/doctorProfile/DoctorProfile.page.jsx:166]: `appointments.length`

File: src/pages/profiles/components/userProfile/UserProfile.page.jsx [hunk 2]
```diff
@@ -73,73 +75,83 @@ export const UserProfile = ({
...
-        <Button
-          variant="solid"
-          color="info"
-          className={`${editState} save__button`}
-          onClick={handleEditOff}
-        >
-          Save changes
-        </Button>
+        <div className="appointments__section">
+          {appointments.length > 0 ? (
```
Comment (ht1204) [inline src/pages/profiles/components/userProfile/UserProfile.page.jsx:149]: `appointments.length`

File: src/pages/profiles/components/userProfile/UserProfile.page.jsx [hunk 3]
```diff
@@ -73,73 +75,83 @@ export const UserProfile = ({
...
+            <AppontmetsList appointments={appointments} />
```
Comment (ht1204) [inline src/pages/profiles/components/userProfile/UserProfile.page.jsx:150]: `AppointmentList`

Comment (ht1204) [reply src/pages/profiles/components/userProfile/UserProfile.page.jsx:150]: Consider a renaming fix of this component.

File: src/pages/profiles/components/userProfile/components/Appointments.userProfile.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import {
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Box,
+} from '@chakra-ui/react';
+
+export const AppontmetsList = () => {
```
Comment (ht1204) [inline src/pages/profiles/components/userProfile/components/Appointments.userProfile.jsx:10]: Here.

**Comment by Salvarez0212** [comment]
typos fixed!

File: src/pages/profiles/components/userProfile/UserProfile.page.jsx [hunk 5]
```diff
@@ -146,8 +147,8 @@ export const UserProfile = ({
         </div>
 
         <div className="appointments__section">
-          {appointments.length > 0 ? (
-            <AppontmetsList appointments={appointments} />
+          {appointments.length ? (
+            <AppointmetsList appointments={appointments} />
```
Comment (ht1204) [inline src/pages/profiles/components/userProfile/UserProfile.page.jsx:151]: `AppointmentsList`

File: src/pages/profiles/components/userProfile/components/Appointments.userProfile.jsx [hunk 6]
```diff
@@ -7,7 +7,7 @@ import {
   Box,
 } from '@chakra-ui/react';
 
-export const AppontmetsList = () => {
+export const AppointmetsList = () => {
```
Comment (ht1204) [inline src/pages/profiles/components/userProfile/components/Appointments.userProfile.jsx:10]: `AppointmentsList`

**Pull request merged**